data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,77 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	swizzle 6
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		"cost" 25100000
+		"shields" 37200
+		"hull" 11780
+		"required crew" 62
+		"bunks" 70
+		"mass" 725
+		"drag" 9.5
+		"heat dissipation" .4
+		"fuel capacity" 500
```

Comment:
The Unfettered would not waste scarce research resources on a one-off build for their leader's ego. This would have to be a prototype for a possible Shield Beetle replacement. (Possibly an abandoned prototype, but not an intentional one-off.)

An improved Beetle hull (not design) should solve fix the Shield Beetle's weakness: low heat capacity, no ramscoop, too little fuel, and lack of armor. We added the obsolete Ladybug design to explain the Unfettered's ability to fix the first three problems without significant research or new inventions.

Any improvements should not come at the cost of the Hai's greatest strengths: fast, maneuverable ships.

Reply:
My thinking is that they added more of the same armor, and the Ladybug tech improved the heat dissipation. The Ladybug tech, uses lots of fins to dissipate heat more easily (similar to Osprey).  The built-in ramscoop comes from gills on the ship (they're in the back near the engines). Most of these ships won't have Fuel Processors because those are in short supply. 

The paper-thin hull is more of a problem than the shields, and a larger heat capacity reduces the need for cooling. 

The more jagged your metal structure, the better it will dissipate heat, since there is a larger surface area. Also, metal (which armor is typically made of) has a low heat capacity, so it will dissipate heat quickly. The Osprey designed was based on that. Same with CPU heat sinks. The "Emperor Beetle" is much less smooth than the Shield Beetle, so it will benefit as well.

Reply:
The Uhai experiment with differents ship modifications for now knowing full well they cant make and produce a new design yet